_locales/en/messages.json

```diff
@@ -2336,6 +2336,9 @@
     "osdElement_MAIN_BATT_VOLTAGE": {
         "message": "Battery Voltage"
     },
+    "osdElement_SAG_COMP_MAIN_BATT_VOLTAGE": {
```

Comment:
Maybe we should add `osdElement_SAG_COMP_MAIN_BATT_VOLTAGE_HELP` message to explain what the sag compensated voltage means?